app/controllers/application_controller.rb

```diff
@@ -1851,10 +1856,30 @@ def identify_tl_or_perf_record
     identify_record(params[:id], controller_to_model)
   end
 
-  def assert_privileges(*features)
-    if features.none? { |feature| role_allows?(:feature => feature) }
-      raise MiqException::RbacPrivilegeException, _('The user is not authorized for this task or item.')
-    end
+  def assert_privileges(*features, any: false)
+    msg = "Features checked: #{features.join(', ')}"
+
+    pass =
+      if any
+        role_allows?(:feature => features.first, :any => true)
+      else
+        features.any? { |feature| role_allows?(:feature => feature) }
```

Comment:
Don't ask about this.  I want to eliminate this conditional.  I didn't want to perform surgery on all the places passing `any`.  It's not many but didn't want to risk breaking it here.

Reply:
😉